low-level/files/opt/seagate/sspl/bin/feed_sspl_conf_to_consul.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+"""
+ ******************************************************************************
```

Comment:
Why cant we avoid consul use here? Its just to reuse existing code & flow and achieve self provisioning with minimal set of changes ? But eventually we need to do away with consul direct use, so why not and start using conf from file directly.
Probably am missing some discussion point for which we still kept this, please remind me.

Reply:
`why not and start using conf from file directly` New commit carries this and consul dependency is removed.
This file will be handled in SSPL Setup PR.